code-of-conduct.md

```diff
@@ -0,0 +1,105 @@
+- Other conduct, which could reasonably be considered inappropriate in a professional setting
+
+## Our responsibilities
+
+Project maintainers are responsible for clarifying and enforcing our standards of acceptable behavior and will take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this code of conduct, and will communicate reasons for moderation decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces, and also applies when an individual is officially representing the project or its community in public spaces. Examples of representing our community include using an official e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the project team responsible for enforcement at **legal@wso2.com**. All complaints will be reviewed and investigated promptly and fairly.
+
+All project/community leaders are obligated to respect the privacy and security of the reporter of any incident.
+
+## Enforcement guidelines
+
+Community leaders will follow the below Community Impact Guidelines in determining the consequences for any action they deem in violation of this code of conduct:
+
+**1. Correction**
+
+<table border=1>
+<th> Community Impact </th>
+<th> Consequence </th>
+<tr border=1>
+<td width=50% border=1> Use of inappropriate language or other behavior deemed unprofessional or unwelcome in the community. </td>
+<td width=50% border=1> A private, written warning from community leaders providing clarity around the nature of the violation and an explanation of why the behavior was inappropriate. A public apology may be requested.
+</td>
+</tr>
+</table>
+
+**2. Warning**
+
+<table border=1>
+<th> Community Impact </th>
+<th> Consequence </th>
+<tr>
+<td width=50%> A violation through a single incident or series of actions. </td>
+<td width=50%>> A warning with consequences for continued behavior. No interaction with the people involved including unsolicited interaction with those enforcing the code of conduct for a specified period of time. This includes avoiding interactions in community spaces as well as external channels like social media. Violating these terms may lead to a temporary or permanent ban.
+</td>
+</tr>
+</table>
+
+**3. Temporary ban**
+
+<table border=1>
+<th> Community Impact </th>
+<th> Consequence </th>
+<tr>>
+<td width=50%>> A serious violation of community standards including sustained inappropriate behavior. </td>
```

Comment:
```suggestion
<td width=50%> A serious violation of community standards including sustained inappropriate behavior. </td>
```

Reply:
Check this in all the places. Seems not fixed in all the places

Reply:
Fixed.